test/TestUtils/project.json

```diff
@@ -17,20 +17,17 @@
       ]
     }
   },
-
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-4338"
+      "version": "1.0.0"
     },
```

Comment:
I put this to stable 1.0.0 because of the others but I can attempt to make it consume the one being built but I suspect it also runs on stage0.